.gitlab-ci.yml

```diff
@@ -654,10 +654,30 @@ library:ci-argosy:
 library:ci-bbv:
   extends: .ci-template
 
+library:ci-coqutil:
+  extends: .ci-template-flambda
+
+library:ci-kami:
+  extends: .ci-template-flambda
+
+library:ci-coq_record_update:
+  extends: .ci-template-flambda
+
+library:ci-riscv_coq:
+  extends: .ci-template
+  stage: stage-3
+  needs:
+    - library:ci-coqutil
```

Comment:
If you set `needs` you must include the build job in the list (see eg ci-color below)